### PR TITLE
day1-day2

### PR DESCRIPTION
**Pull request by sujit-thinkitive**

### CODE REVIEW

File: JPA-and-Hibernate/.mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.5/apache-maven-3.9.5-bin.zip
```
Comment (yogesh-think) [inline JPA-and-Hibernate/.mvn/wrapper/maven-wrapper.properties:1]: Add this file in .gitignore file

File: JPA-and-Hibernate/mvnw [hunk 2]
```diff
@@ -0,0 +1,308 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
```
Comment (yogesh-think) [inline JPA-and-Hibernate/mvnw:10]: Add this file in .gitignore file

File: JPA-and-Hibernate/mvnw.cmd [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+@REM regarding copyright ownership.  The ASF licenses this file
+@REM to you under the Apache License, Version 2.0 (the
+@REM "License"); you may not use this file except in compliance
+@REM with the License.  You may obtain a copy of the License at
+@REM
+@REM    https://www.apache.org/licenses/LICENSE-2.0
+@REM
+@REM Unless required by applicable law or agreed to in writing,
+@REM software distributed under the License is distributed on an
+@REM "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
```
Comment (yogesh-think) [inline JPA-and-Hibernate/mvnw.cmd:14]: Add this file in .gitignore file

File: project/src/main/java/com/firstStringBoot/project/example1/CourseController.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RestController
+public class CourseController {
+
+    @RequestMapping("/courses")
+    public List<Course> retrieveAllCourses(){
```
Comment (yogesh-think) [inline project/src/main/java/com/firstStringBoot/project/example1/CourseController.java:13]: Implement example that different from the tutorial.

File: JPA-and-Hibernate/.mvn/wrapper/maven-wrapper.properties (hunk 1 above)
Comment (sujit-thinkitive) [reply JPA-and-Hibernate/.mvn/wrapper/maven-wrapper.properties:1]: i will update it in .gitignore

File: JPA-and-Hibernate/mvnw (hunk 2 above)
Comment (sujit-thinkitive) [reply JPA-and-Hibernate/mvnw:10]: i will update it in .gitignore

File: JPA-and-Hibernate/mvnw.cmd (hunk 3 above)
Comment (sujit-thinkitive) [reply JPA-and-Hibernate/mvnw.cmd:14]: i will update it in .gitignore

File: project/src/main/java/com/firstStringBoot/project/example1/CourseController.java (hunk 4 above)
Comment (sujit-thinkitive) [reply project/src/main/java/com/firstStringBoot/project/example1/CourseController.java:13]: Okey i will use different examples

**Pull request merged**